README.md

```diff
@@ -35,7 +36,7 @@ $pairManager = $this->get("tbbc_money.pair_manager");
 $usd = $pairManager->convert($tenEur, 'USD');
 
 // a form integration
-$formBuilder->add("price", "tbbc_money");
+$formBuilder->add("price", MoneyType::class);
```

Comment:
Not sure this is a good idea to show a syntax that won't work for those still using php 5.3